postgres-appliance/Dockerfile.build

```diff
@@ -279,7 +279,13 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         && if [ -d /usr/share/postgresql/9.5/contrib/postgis-$POSTGIS_VERSION ]; then \
             cd /usr/share/postgresql/9.5/contrib/postgis-$POSTGIS_VERSION \
             && for f in *.sql *.pl; do \
-                for v in 9.4 9.6 10; do \
+                 if [ -L /usr/share/postgresql/9.5/contrib/postgis-$POSTGIS_VERSION/$f ]; then \
+                   continue; \
+                 fi; \
+                 for v in $PGOLDVERSIONS; do \
+                     if [ $v == 9.5 ]; then \
```

Comment:
I think it would be better to combine it with the next if block:
`if [ $v != 9.5 ] && [ -f /usr/share/postgresql/$v/contrib/postgis-$POSTGIS_VERSION/$f ] \`